### ISSUE TITLE
Build problem with latest cesm-coupling branch

### ISSUE THREAD

**Issue by ekluzek** [ESCOMP/mizuRoute#330]
I'm getting a build error with the latest cesm-coupling branch on cheyenne_intel, as follows...

The git hash is: f669e9775b04592352c37b996a997e000a653990

```
         - Building rof Library 
Building rof with output to /glade/scratch/erik/SMS_P360x1_Lm1.f09_f09_rHDMAlk_mg17.I2000Clm50SpMizGs.cheyenne_intel.mizuroute-default.20221123_225752_dv160r/bld/rof.bldlog.221123-230903
datm built in 1.160864 seconds
/glade/work/erik/ctsm_worktrees/mizuRoute/components/mizuRoute/route/build/cpl/RtmMod.F90(357): error #7002: Error in opening the compiled module file.  Check INCLUDE paths.   [NR_UTILITY_MODULE]

/glade/work/erik/ctsm_worktrees/mizuRoute/components/mizuRoute/route/build/cpl/RtmMod.F90(357): error #6580: Name in only-list does not exist or is not accessible.   [ARTH]

mizuroute built in 22.193748 seconds
ERROR: BUILD FAIL: mizuroute.buildlib failed, cat /glade/scratch/erik/SMS_P360x1_Lm1.f09_f09_rHDMAlk_mg17.I2000Clm50SpMizGs.cheyenne_intel.mizuroute-default.20221123_225752_dv160r/bld/rof.bldlog.221123-230903
```

**Comment by ekluzek** [comment]
I think the fix is just to do this...

``` fortran
diff --git a/route/build/cpl/RtmMod.F90 b/route/build/cpl/RtmMod.F90
index 9ce2a0a..1576502 100644
--- a/route/build/cpl/RtmMod.F90
+++ b/route/build/cpl/RtmMod.F90
@@ -354,7 +354,7 @@ SUBROUTINE route_run(rstwr)
     USE write_restart_pio,   ONLY: restart_output   ! subroutine: write out restart file
     USE init_model_data,     ONLY: update_time      ! subroutine: increment time
     USE process_remap_module,ONLY: basin2reach      ! subroutine: mapping variables in hru domain to reach domian
-    USE nr_utility_module,   ONLY: arth             ! utility: equivalent to python range function
+    USE nr_utils,            ONLY: arth             ! utility: equivalent to python range function
 
     implicit none
     ! Arguments:
```

**Comment by ekluzek** [comment]
This was fixed #329
